### PR TITLE
Modify the path to the pipeline scripts

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This commit:
- Modifies the readme to reflect that the nodevertical scale test
  is managed by scale-ci-watcher.
- Modifies the path to the pipeline scripts to be loaded now that
  we are using scale-ci-pipeline under openshift-scale org instead
  of openshift/aos-cd-jobs.
- Modifies the scale-ci-pipeline template to use the master branch.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -71,62 +71,62 @@ node (node_label) {
...
 	if (byo == "TRUE") {
-		load "scale-ci/pipeline-scripts/byo.groovy"
+		load "pipeline-scripts/byo.groovy"
 	}
 
 	// stage to run OCP 4.X scaleup
-	if (ocp_4.x_scaleup == "TRUE") {
-		load "scale-ci/pipeline-scripts/OCP-4.X/scaleup.groovy"
+	if (ocp4_scaleup == "TRUE") {
+		load "pipeline-scripts/"
```
Comment (akrzos) [inline Jenkinsfile]: Should this have a specific file it is linked to?

Comment (chaitanyaenr) [reply Jenkinsfile]: It should point to the scaleup_4.x groovy script, fixed it. Thanks.

**Comment by akrzos** [comment]
@chaitanyaenr looks like a merge conflict needs to be resolved.

**Comment by chaitanyaenr** [comment]
@akrzos resolved the merge conflict, thanks.

**Pull request merged**